search-app/src/cmr/search/validators/all_granule_validation.clj

```diff
@@ -58,14 +58,14 @@
 (defn no-all-granules-with-scroll
   "Validates that the query is not an all granules query if it is a scroll query."
   [query]
-  (when (and (:scroll query) 
-             (all-granules-query? query) 
-             
+  (when (and (:scroll query)
+             (all-granules-query? query)
+             (not= :created-at (get-in query [:condition :field]))
```

Comment:
We need to talk about this a little bit more. I don't think we want to allow an all granules query for scrolling at all. The user would first submit a query to see which collections have updated granules, then provide the granule query with a list of concept-ids and scroll through those results. Let's talk about it tomorrow.